crates/adapter/src/aptos_writer/contracts_caller/verify_fri/extract_verify_fri_input.rs

```diff
@@ -5,18 +5,15 @@ use aptos_sdk::move_types::value::MoveValue;
 
 use crate::aptos_writer::contracts_caller::verify_fri::types::fri_verify_input::FriVerifyInput;
 
-pub fn extract_verify_fri_input(
-    fri_inputs: &[String],
-) -> anyhow::Result<
-    Vec<(
-        FriVerifyInput,
-        MoveValue,
-        MoveValue,
-        MoveValue,
-        MoveValue,
-        MoveValue,
-    )>,
-> {
+type FriVerifyInputVec = Vec<(
```

Comment:
(
     FriVerifyInput,
     MoveValue,
     MoveValue,
     MoveValue,
     MoveValue,
     MoveValue,
 ) => to a new struct